File: demos-text-classifier/src/main/python/model/local_government.py
from model.aggregate_root import AggregateRoot


class LocalGovernment(AggregateRoot):

    def __init__(self,
                 ident: str = '',
                 name: str = '',
                 domain_name: str = '',
                 official_council_reports: set = set()):
        self.id = ident
        self.name = name
        self.domain_name = domain_name
        self.official_council_reports = official_council_reports

    def get_id(self):
        return self.id

    def set_id(self, ident: str):
        self.id = ident

File: demos-model/src/main/java/org/demos/model/domains/localgovernment/LocalGovernment.java
package org.demos.model.domains.localgovernment;

import java.util.Map;

public interface LocalGovernment {

    public Long getId();

    public void setId(Long id);

    public String getWebSite();

    public void setWebSite(String webSite);

    public String getName();

    public void setName(String name);

    public Float getLatitude();

    public void setLatitude(Float latitude);

    public Float getLongitude();

    public void setLongitude(Float longitude);

    public String getCodification();

    public void setCodification(String codification);

    public LocalGovernmentType getType();

    public void setType(LocalGovernmentType type);

    public String getZipCode();

    public void setZipCode(String zipCode);
}
